src/Makefile.am

```diff
@@ -525,6 +526,7 @@ libbitcoin_util_a_SOURCES = \
   util/time.cpp \
   util/url.cpp \
   util/validation.cpp \
+  util/asmap.cpp \
```

Comment:
Any reason this is placed last instead of in alphabetical order?

Reply:
It was already messed up before me... But alright, will put my file in a proper place.